Makefile

```diff
@@ -84,4 +84,4 @@ test-failpoint:
 .PHONY: test-robustness # Running robustness tests requires root permission
 test-robustness:
 	go test -v ${TESTFLAGS} ./tests/dmflakey -test.root
-	go test -v ${TESTFLAGS} ./tests/robustness -test.root
+	go test -v ${TESTFLAGS} ${ROBUSTNESS_TESTFLAGS} ./tests/robustness -test.root
```

Comment:
I notice we don't have any graceful validation that whoever runs `make test-robustness` has actually run the pre-requisite `make gofail-enable` prior.  Currently we just get cryptic error printed which folks might not understand at first glance, i.e:

```bash
=== RUN   TestRestartFromPowerFailure/fp_ext4_commit5s                                         
    powerfailure_test.go:112: start bbolt bench -work -path /tmp/TestRestartFromPowerFailurefp_ext4_commit5s797232614/002/boltdb -count=1000000000 -batch-size=5
    powerfailure_test.go:129: simulate power failure                                           
    powerfailure_test.go:134: random pick failpoint: resizeFileError                           
    powerfailure_test.go:168:                                                                  
                Error Trace:    /home/james/Documents/bbolt/tests/robustness/powerfailure_test.go:168
                                                        /home/james/Documents/bbolt/tests/robustness/powerfailure_test.go:135
                                                        /home/james/Documents/bbolt/tests/robustness/powerfailure_test.go:81
                Error:          Received unexpected error:                                     
                                Put "http://127.0.0.1:12345/resizeFileError": dial tcp 127.0.0.1:12345: connect: connection refused
```

In `etcd-io/etcd` we introduced https://github.com/etcd-io/etcd/blob/f8d5ba9a3fdff4f706f06ef0f0c0a27bb033ac53/tests/framework/integration/testing.go#L102. Perhaps we should introduce something similar here, not a blocker on this pr though.

Reply:
I build bbolt with failpoints. I think gofail.List doesn't work for this case. Let me think about how to handle.